MrFish.lua

```diff
@@ -266,13 +267,13 @@ function addon:StartFishFrame(atCursor)
 	start.Amount:SetFormattedText(TRADESKILL_RANK_WITH_MODIFIER,fishingSkill,fishingBonus,fishingCap)
 end
 function addon:StopFishFrame(show)
-	local body,main,off='/stopcasting',weapons[INVSLOT_MAINHAND].name,weapons[INVSLOT_OFFHAND].name
+	local body,main,off='/stopcasting',weapons[INVSLOT_MAINHAND].link,weapons[INVSLOT_OFFHAND].link
 	if (main and off) then
-			body=format("/stopcasting\n/equip %s\n/equip %s",main,off)
+		body=format("/stopcasting\n/equipslot 16 %s\n/equipslot 17 %s",main,off)
 	elseif (main or off) then
-			body=format("/stopcasting\n/equip %s",main or off)
+		body=format("/stopcasting\n/equipslot 16 %s",main or off)
```

Comment:
Please, do not use hardcoded values for inventory/equipment slot

Reply:
see comment above